txtwitter/tests/fake_twitter.py

```diff
@@ -244,6 +257,33 @@ def to_dict(self, twitter_data):
         return user_dict
 
 
+class FakeMedia(object):
+    def __init__(self, media_id_str, fake_image, **kw):
+        self.media_id_str = media_id_str
+        self.size = fake_image.size
+        self.expires_after_secs = 60
+        self.image = {
+            'image_type': 'image/jpeg',
+            'w': fake_image.width,
+            'h': fake_image.height,
+        }
+
```

Comment:
My thought was that, as long as we're only uploading images, we can expect the response from the Twitter API to be consistent each time, but I guess that's a little short sighted considering we'd like to keep using these entities if the client ever supports uploading of other media types. I'll keep the `kw` rather.